deployment/helm/ditto/values.yaml

```diff
@@ -1376,6 +1376,13 @@ thingsSearch:
       updaterPersistenceReadConcern: "local"
       # updaterPersistenceReadPreference configures the MongoDB read preference for the "ThingUpdater"
       updaterPersistenceReadPreference: "primaryPreferred"
+      updaterPersistenceThrottling:
```

Comment:
I would also rename the config accordingly to be more specific what is throttled.

And - the `updaterPersistenceThrottling` does not yet have a documentation line.